script/style/checkstyle.xml

```diff
@@ -79,6 +79,8 @@
         </module>
 
         <module name="VisibilityModifier">
+            <property name="packageAllowed" value="true"/>
```

Comment:
Did you try setting packageAllowed to false, not true? On my machine it somehow worked.